releases/cloud/point-2024-02-22-release-note.md

```diff
@@ -0,0 +1,47 @@
+---
+description: >-
+  This article lists improvements and bug fixes in the Syskit Point Cloud version 2024.1.43.10
+---
+
+# February 22, 2024
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business.
+
+
+## Improvements & Bug Fixes
+
+* **The Change Storage Limit action** was added to the Storage Metrics report.
```

Comment:
ovo je korisna akcija
trebalo bi:
- malo detaljnjije raspisati u Release noteu što ta akcija radi
- možda prebaciti pod feature
- prijaviti task da se akcija opiše i na drugim člancima u storage article-ima